### PR TITLE
Add Drain Cleaner 1.1.0 to the installation files

### PR DESCRIPTION
**Pull request by scholzj**
### Type of change

- Task

### Description

This PR adds the Drain Cleaner 1.1.0 release to the installation files and updates the docs and STs accordingly.

### Checklist

- [x] Make sure all tests pass
- [x] Update documentation
- [x] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [x] Update CHANGELOG.md

### CODE REVIEW

**Review by ppatierno — APPROVED** [review]

**Review by im-konge — APPROVED** [review]
Thanks :)

**Review by see-quick — APPROVED** [review]

File: documentation/modules/drain-cleaner/proc-drain-cleaner-deploying.adoc [hunk 1]
```diff
@@ -72,7 +73,8 @@ spec:
 
 .Procedure
 
-. Set `maxUnavailable` to `0` (zero) in the Kafka and ZooKeeper sections of the `Kafka` resource using `template` settings.
+. If you use the legacy mode activated by setting the `STRIMZI_DENY_EVICTION` environment variable to `false`, you have to configure the `PodDisruptionBudget` resource.
```
Comment (PaulRMellor) [inline documentation/modules/drain-cleaner/proc-drain-cleaner-deploying.adoc:76]:
```suggestion
. If you are using the legacy mode activated by setting the `STRIMZI_DENY_EVICTION` environment variable to `false`, you must also configure the `PodDisruptionBudget` resource.
```

**Review by PaulRMellor — APPROVED** [review]
Much simpler now. Thanks for the doc updates

**Pull request merged**